pressurecooker/images.py

```diff
@@ -116,8 +179,35 @@ def create_waveform_image(fpath_in, fpath_out, max_num_of_points=None, colormap_
 
         # Plot points
         ax.plot(np.arange(count), subsignals, color)
-        ax.set_aspect('auto')
-
+        ax.set_aspect("auto")
         canvas.print_figure(fpath_out)
+
     finally:
         os.remove(tempwav_name)
+
+
+
+# TILED THUMBNAILS FOR TOPIC NODES (FOLDERS)
+################################################################################
+
+def create_tiled_image(source_images, fpath_out):
+    """
+    Create a 16:9 tiled image from list of image paths provided in source_images
+    and write result to fpath_out.
+    """
+    sizes = {1:1, 4:2, 9:3, 16:4, 25:5, 36:6, 49:7}
+    assert len(source_images) in sizes.keys(), "Number of images must be a perfect square <= 49"
```

Comment:
Have you thought about taking the first X photos where X is greatest perfect square less than or equal to `len(source_images)`?

Reply:
The calling code sends in either 1 or 4 images depending if there is enough:
https://github.com/learningequality/ricecooker/blob/master/ricecooker/classes/files.py#L834-L840
so we don't have to worry about that.

I wonder if what the case 3:9 woudl look like.... probably too crowded but would be nice to say "hey there are lots of stuff in this folder"